lib/index.js

```diff
@@ -38,11 +38,20 @@ exports.plugin = {
 
           case 'insertone': {
             const inserted = await db.collection(collection).insertOne(request.payload)
+            if (setTimestamps) {
```

Comment:
You are setting this too late as the insert call has already happened. Can you also simplify with defining a variable that holds `new Date().getTime()`

Reply:
Fixed this